faker/providers/file/__init__.py

```diff
@@ -269,16 +282,28 @@ def file_path(
         If ``absolute`` is ``True`` (default), the generated path starts with
         ``/`` and is absolute. Otherwise, the generated path is relative.
 
+        if ``file_system`` is set (default="linux"), the generated path uses
+        specified file system path standard, the list of valid file systems include:
+        ``'windows'``, ``'linux'``.
+
         :sample: size=10
         :sample: depth=3
         :sample: depth=5, category='video'
         :sample: depth=5, category='video', extension='abcdef'
+        :sample: depth=5, category='video', extension='abcdef', file_system='windows
```

Comment:
typo (missing closing quote):

```suggestion
        :sample: depth=5, category='video', extension='abcdef', file_system='windows'
```